website/src/components/Messages/MessageTableEntry.tsx

```diff
@@ -158,6 +168,25 @@ export function MessageTableEntry({
           <LabelMessagePopup message={message} show={labelPopupOpen} onClose={closeLabelPopup} />
           <ReportPopup messageId={message.id} show={reportPopupOpen} onClose={closeReportPopup} />
         </HStack>
+        <Flex position="absolute" gap="2" top="-2.5" right="5">
```

Comment:
can you use `start` instead of `right`? for RTL languages?

Reply:
I'm tried insetEnd but it didn't work? Could you help me with rtl stuff?

Reply:
yeah I guess it should be `inset-block-end` or maybe `inset-inline-end`, if it does not work you can leave like this, we need to fix a lot of RTL stuff anyway.

Reply:
Both of them didn't work. Charka only render right. Charka [docs ](https://chakra-ui.com/docs/styled-system/rtl-support) require to install `stylis` and `stylis-plugin-rtl`. I think we should ignore them for now